src/controlllers/AgentSignup.js

```diff
@@ -0,0 +1,44 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import Model from '../models/model';
+
+const agentModel = new Model('agents');
+
+export const createAgent = async (req, res) => {
+  const saltRounds = 5;
+  const {
+    firstName, lastName, email, password, phoneNo
+  } = req.body;
+  const hashedPassword = await bcrypt.hash(password, saltRounds);
+  const columns = 'first_name, last_name, email, password, phone_no';
+  const values = `'${firstName}', '${lastName}', '${email}', '${hashedPassword}', '${phoneNo}'`;
+  try {
+    const validEmail = await agentModel.select('*', ` WHERE  email = '${email}' `);
+    if (validEmail.rows.length > 0) {
+      return res.status(409).json({ messages: 'Email already Exist' });
+    }
+    const data = await agentModel.insertWithReturn(columns, values);
+    const token = jwt.sign({ id: data.rows.id }, process.env.TOKEN_KEY, {
+      expiresIn: '1m',
```

Comment:
`1 minute` is too short for an expiry time of a token, I can suggest you set it in hours or 2 days.